config/profile/rhdh/plugin-deps/sonataflow.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-knative-to-sonataflow-and-workflows # hardcoded
+spec:
+  podSelector: {}
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              # Allow knative events to be delivered to workflows.
+              kubernetes.io/metadata.name: knative-eventing
+        - namespaceSelector:
+            matchLabels:
+              # Allow auxiliary knative function for workflow (such as m2k-save-transformation)
+              kubernetes.io/metadata.name: knative-serving
+---
+# NetworkPolicy to unblock incoming traffic to the namespace
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-external-communication # hardcoded
+spec:
+  podSelector: {}
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              # Allow knative events to be delivered to workflows.
+              policy-group.network.openshift.io/ingress: ""
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-intra-network # hardcoded
+spec:
+  # Apply this policy to all pods in the namespace
+  podSelector: {}
+  # Specify policy type as 'Ingress' to control incoming traffic rules
+  policyTypes:
+    - Ingress
+  ingress:
+    - from:
+        # Allow ingress from any pod within the same namespace
+        - podSelector: {}
+---
+# NetworkPolicy to allow openshift-user-workload-monitoring pods to access all pods within the workflow's namespace
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-monitoring-to-sonataflow-and-workflows # hardcoded
+spec:
+  # Apply this policy to all pods in the namespace
+  podSelector: {}
+  # Specify policy type as 'Ingress' to control incoming traffic rules
+  policyTypes:
+    - Ingress
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              # Allow openshift-user-workload-monitoring pods to access the workflow.
+              kubernetes.io/metadata.name: openshift-user-workload-monitoring
+---
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  Registry: {}
+---
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  controller-custom-certs:
+    name: ""
+    type: ""
+  registry: {}
```

Comment:
These are cluster-wide resources. Shouldn't they be moved to `plugin-infra`, similar to what is done in the orchestrator-infra Helm Chart? https://github.com/redhat-developer/rhdh-chart/blob/main/charts/orchestrator-infra/templates/serverless/knatives.yaml#L28-L54
(I guess you would have the same issue with the CRDs then..)

Reply:
They are cluster wide but not sure they can be considered as "infra". 
Can we discuss it separately, I think it can wait for next PRs ?